### PR TITLE
Newmacros

### PR DESCRIPTION
**Pull request by Casteran**
Add xspace to @Zimmi48  PR 92
(Note: some macros are still partly used : `\alectr`, `gaia`, etc. To do mañana.

### CODE REVIEW

File: doc/JFLA2022/paper.tex [hunk 1]
```diff
@@ -59,6 +59,8 @@
 \newcommand{\texniccenter}{{\TeX}nicCenter}
 \newcommand{\makefile}{\texttt{Makefile}}
 \newcommand{\latexeditor}{LEd}
+\newcommand{\Hydras}{Hydras \& Co.\xspace}
+\newcommand{\Coqcommunity}{\emph{Coq-community\xspace} }
```
Comment (palmskog) [inline doc/JFLA2022/paper.tex:63]:
```suggestion
\newcommand{\Coqcommunity}{Coq-community\xspace}
```
I think we want to put the `\emph{...}` where we actually use the macro, for more flexibility. Do we really want Coq-community emphasized everywhere? I think this is what is causing the issues with `~\cite{XY}`.

File: doc/JFLA2022/paper.tex [hunk 2]
```diff
@@ -150,7 +151,8 @@ \subsection{Background}
 \end{itemize}
 
 \subsection{Vision}
-The \emph{Hydra-battles} project, part of \community~\cite{CoqCommunity} aims to be an experimental platform for the collaborative development of commented libraries of formal proofs. \community is a community organization that we have founded in 2018 with two goals in mind: providing a solution for the long-term maintenance of interesting \coq packages, and working collaboratively on documentation projects. The \emph{Hydra-battles} project demonstrates that these two goals are not independent: interesting \coq packages can become the basis for new documentation.
+The \Hydras project, part of \Coqcommunity\cite{CoqCommunity} aims to be an experimental platform for the collaborative development of commented libraries of formal proofs. \Coqcommunity is a community organization that we have founded in 2018 with two goals in mind: providing a solution for the long-term maintenance of interesting Coq packages, and working collaboratively on documentation projects. The \Hydras project demonstrates that these two goals are not independent: interesting Coq packages can become the basis for new documentation.
```
Comment (palmskog) [inline doc/JFLA2022/paper.tex:154]:
```suggestion
The \Hydras project, part of \Coqcommunity~\cite{CoqCommunity} aims to be an experimental platform for the collaborative development of commented libraries of formal proofs. \Coqcommunity is a community organization that we have founded in 2018 with two goals in mind: providing a solution for the long-term maintenance of interesting Coq packages, and working collaboratively on documentation projects. The \Hydras project demonstrates that these two goals are not independent: interesting Coq packages can become the basis for new documentation.
```
If you apply the fix to `\Coqcommunity` macro, we should be able to use the standard `~\cite{XY}`.

File: doc/JFLA2022/paper.tex (hunk 1 above)
Comment (Casteran) [reply doc/JFLA2022/paper.tex:63]: I agree with with your suggested change.  If we want to emphasize for instance the first occurrence of Coq-community, in the paper, it is still possible to write `\emph{\Coqcommunity}` . Is it usual ?

Comment (palmskog) [reply doc/JFLA2022/paper.tex:63]: Yes, I think writing `\emph{\Coqcommunity}` is the standard way to do it.

**Review by palmskog — APPROVED** [review]

**Pull request merged**